language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/docs/DocumentationGenerator.java

```diff
@@ -285,9 +295,25 @@ private static DocAttachmentInfo getFunctionNodeDocumentation(FunctionSignatureN
             paramName.ifPresent(token -> parameters.put(token.text(), "Parameter Description"));
         });
         String returnDesc = signatureNode.returnTypeDesc().isPresent() ? "Return Value Description" : null;
-        if (hasDeprecated) {
-//            attributes.add(getDeprecatedDescription(offset));
+//        String deprecated = null;
+//        if (hasDeprecated) {
+//            deprecated = "This function is deprecated.";
+//        }
+        return new DocAttachmentInfo(desc, parameters, returnDesc, docStart,
+                                     getPadding(signatureNode.parent(), syntaxTree));
+    }
+
+    private static String getPadding(NonTerminalNode bLangFunction, SyntaxTree syntaxTree) {
+        LinePosition position = bLangFunction.location().lineRange().startLine();
+        TextDocument textDocument = syntaxTree.textDocument();
+        int prevCharIndex = textDocument.textPositionFrom(LinePosition.from(position.line(), position.offset()));
+        int lineStartIndex = textDocument.textPositionFrom(LinePosition.from(position.line(), 0));
+        String sourceCode = syntaxTree.toSourceCode();
+        String padding = sourceCode.substring(lineStartIndex, prevCharIndex);
+        if (padding.isBlank()) {
+            return padding;
+        } else {
+            return " ";
         }
```

Comment:
Minor Suggestion: May be we can collapse to a ternary expression since the check is simple

Reply:
Yes can convert to a ternary. will do